it/lessons/basics/basics.md

```diff
@@ -91,6 +111,20 @@ iex> :true === true
 true
 ```
 
+I Nomi dei moduli in Elixir sono achessi atomi. `MyApp.MyModule` è un atom valido, anche se nessun modulo con questo nome è stato dichiarato.
```

Comment:
Fix `achessi` with `anch'essi`.

Also, not sure if `atomi` is the right translation. I mean, an `atom` is a word/concept that maybe should not be translated at all.

Reply:
I do agree with you here, `atom` shouldn't be translated, but I saw in other lessons that the word `tuple` gets translated to `tupla` (singular) and `tuple` (plural). And I try to be consistent.

Is there a rule here? I think we shouldn't translate any of the specific elixir/erlang or any technical term at all. What do you say? :)

Reply:
@gemantzu @doomspork also what do you think here?

Reply:
maybe `tuple`/`tupla` is a bit different. I'm not sure if it exists in italian language (that would explain why it gets translated). you should ask to someone who did CS in Italy :-P